### PR TITLE
added example for redirecting apex domain, fixed bugs in ZH version

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: en/docs/edge-logic/faq.md [hunk 1]
```diff
@@ -156,7 +156,8 @@ Our [CNAME-based traffic management system](</docs/portal/traffic-management/ove
...
 
-Note that each anycast IP address is broadcast only in a small subset of our global PoPs, and the list of PoPs where anycast is available is subject to changes without prior notice. Therefore, the performance and capacity behind an anycast IP address are not comparable to that of an edge hostname. We recommend using the CNAME-based edge hostnames to route traffic as much as possible. Anycast should be used only for apex domains and the property configuration should be a simple redirection to a non-apex domain, such as the www site.
+Note that each anycast IP address is broadcast only in a small subset of our global PoPs, and the list of PoPs where anycast is available is subject to changes without prior notice. Therefore, the performance and capacity behind an anycast IP address are not comparable to that of an edge hostname. We recommend using the CNAME-based edge hostnames to route traffic as much as possible. Anycast should be used only for apex domains and the property configuration should be a simple redirection to a non-apex domain, such as the www site. Here is how to do this on the portal:
```
Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:159]: replace "changes" with "change"
The comma after  "non-apex domain" is not really necessary.
"the www site" sounds strange. Wouldn't it be more appropriate to say "the www subdomain"?

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I made some suggestions.

**Review by Jianghai-Quantil — DISMISSED** [review]
Looks good to me

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**